exercises/concept/train-driver/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+// => [5, 7, 12, 3, 14, 8, 3]
+```
+
+## 2. Move the first two elements to the end of the array
+
+Now that you got a general feel for handling your friend's data. Your friend has noticed that the first two days' values are not in the correct place. 
+Your friend would like you to move the first two days' value to the end of the array.
+
+```exercism/note
+
+Implement a function `fixListOfWagons` that accepts an array of the weight of each wagon.
+It returns an array where the 2 first elements are moved to the end of the array.
+
+```
+
+```javascript
+eachWagonsWieght = [2, 5, 0, 7, 4,  0, 1, 3, 1];
+fixListOfWagons(eachWagonsWieght);
+// => [0, 7, 4,  0, 1, 3, 1, 2, 5]
+```
+
+## 3. Add missing values
+
+Your friend realized that all data wasn't added and found another array which contains the missing values. 
+Your friend would like you to add the missing values to the array. 
+All they can remember is that the missing values should be placed after the first element in the array.
+
+
+```exercism/note
+
+Given this new information, write a function called `CorrectListOfWagons` that takes two arrays which have the values of the weight of each wagon as an argument. 
+The second array should be added after the first element of the first array.
+
+```
+
+
+```javascript
+eachWagonsWieght = [2, 5, 0, 7, 4, 1];
+missingWagons = [3, 0, 6, 1];
+CorrectListOfWagons(eachWagonsWieght, missingWagons);
+// => [2, 3, 0, 6, 1, 5, 0, 7, 4, 1]
+```
+
+## 4. Update routing information
+
+Now that the wagon data is correct, your friend would like you to update the routing information. 
+Your friend has an object with the routing information and would like you to add more routing information to the object. 
+Every route requires a bit different information so your friend would prefer a generic solution.
+
+
+```exercism/note
+
+Implement a function `updateRoutingInformation` that accepts two objects.
+The first object contains which city the train should go between and the second object contains more routing information.
+The function should return an object with the updated routing information.
+
+```
+
+```javascript
+route  = {from: "Berlin", to: "Hamburg"};
+moreRouteInformation = {length: 100, speed: 50};
+updateRoutingInformation(route, moreRouteInformation);
+// => {from: "Berlin", to: "Hamburg", length: 100, speed: 50}
+```
+
+## 5. Remove arrival time from routing information
+
+Your friend has noticed that they don't need the arrival time in the routing information.
+Therefore your friend would like you to remove the arrival time from the routing information.
+
+```exercism/note
+
+Implement a function `removeArrivalTime` that accepts an object with the routing information.
+The function should return an object
+
+``` 
+
+```javascript
+routeInformation= {
+  from: "Berlin",
+  to: "Hamburg",
+  length: 100,
+  timeOfArrival: "10:10"
+};
+removeTimeOfArrival(routeInformation);
+// => {from: "Berlin", to: "Hamburg", length: 100}
+```
```

Comment:
Can we phrase/change this is a way that people are not drawn to simply use "delete"? 🤔

Reply:
I actually thought a bit about this. It is kinda impossible to make this exercise not being solvable with delete. We could change it so you should return time of arrival in its own object. 

```javascript
function removeTimeOfArrival(routeInformation){
   const arrivalTime = {timeOfArrival: routeInformation.routeInformation}
   delete routeInformation.routeInformation
   return arrivalTime, routeInformation
}
```